src/driver/oracle/OracleQueryRunner.ts

```diff
@@ -1491,8 +1492,10 @@ export class OracleQueryRunner extends BaseQueryRunner implements QueryRunner {
      */
     protected createTableSql(table: Table, createForeignKeys?: boolean): Query {
         const columnDefinitions = table.columns.map(column => this.buildCreateColumnSql(column)).join(", ");
+        if(this.driver.reservedWords.includes(table.name.toUpperCase())) {
```

Comment:
Isn't this not as much of a problem if they're escaped via quotes?

Reply:
It's not but same with comment on the change, quotes are not recommended and Oracle stores details in uppercase. I don't believe we should be allowing reserved keywords to be used. From Oracle's site:
```
The following words are reserved by Oracle. That is, they have a special meaning to Oracle and so cannot be redefined. **For this reason, you cannot use them to name database objects such as columns, tables, or indexes.**
```